fuzz/fuzzers/server.rs

```diff
@@ -6,9 +6,47 @@ extern crate rustls;
 use std::io;
```

Comment:
do the corpus files added in this commit have any interesting provenance?

Reply:
They are copies of the existing ones, with 0x00 or 0x01 prepended, so they reach `fuzz_buffered_api` and `fuzz_acceptor_api`.